src/components/data-processing/MitochondrialContent/CalculationConfig.jsx

```diff
@@ -45,7 +45,7 @@ const MitochondrialConfig = (props) => {
           max={10}
           value={config.methodSettings[activeMethod].binStep}
           onUpdate={(newValue) => updateSettingsForActiveMethod({ binStep: newValue })}
-          disabled={disabled}
+          disabled={disabled || plotType === 'logHistogram'}
```

Comment:
why adding `plotType === 'logHistogram'` in here?

Reply:
because we don't need the bin step control on that plot - ticket requirement